modules/FvwmPager/FvwmPager.h

```diff
@@ -169,6 +169,7 @@ RETSIGTYPE DeadPipe(int nonsense);
 void process_message(FvwmPacket*);
 void ParseOptions(void);
 
+void extract_monitor_config(struct fpmonitor *m, char *tline);
```

Comment:
I know it's a mess of styles, but for new additions:

* Scoped functions should be declared as `static`;
* Function prototypes can be unnamed with respect to the variables used.

Hence:

```suggestion
static void extract_monitor_config(struct fpmonitor *, char *);
```